### ISSUE TITLE
Error when starting ipcluster causing jupyter to freeze

### ISSUE THREAD

**Issue by ChristopherMarais** [ipython/ipyparallel#421]
When I try and start the cluster with ipcluster start -n 4 it just sends the following error in the console:

(MIT_807_P38) C:\Users\GCM>ipcluster start -n 4
2020-08-11 16:39:45.147 [IPClusterStart] Starting ipcluster with [daemon=False]
2020-08-11 16:39:45.148 [IPClusterStart] Creating pid file: C:\Users\GCM\.ipython\profile_default\pid\ipcluster.pid
2020-08-11 16:39:45.149 [IPClusterStart] Starting Controller with LocalControllerLauncher
2020-08-11 16:39:45.163 [IPClusterStart] ERROR | Controller start failed
Traceback (most recent call last):
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\ipyparallel\apps\ipclusterapp.py", line 543, in start_controller
    self.controller_launcher.start()
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\ipyparallel\apps\launcher.py", line 339, in start
    return super(LocalControllerLauncher, self).start()
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\ipyparallel\apps\launcher.py", line 269, in start
    self.loop.add_handler(self.stdout, self.handle_stdout, self.loop.READ)
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\tornado\platform\asyncio.py", line 100, in add_handler
    self.asyncio_loop.add_reader(fd, self._handle_events, fd, IOLoop.READ)
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\asyncio\events.py", line 501, in add_reader
    raise NotImplementedError
NotImplementedError
ERROR:tornado.application:Exception in callback functools.partial(<function IPClusterStart.start.<locals>.start at 0x000001D4302A9040>)
Traceback (most recent call last):
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\tornado\ioloop.py", line 743, in _run_callback
    ret = callback()
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\ipyparallel\apps\ipclusterapp.py", line 588, in start
    self.start_controller()
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\ipyparallel\apps\ipclusterapp.py", line 543, in start_controller
    self.controller_launcher.start()
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\ipyparallel\apps\launcher.py", line 339, in start
    return super(LocalControllerLauncher, self).start()
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\ipyparallel\apps\launcher.py", line 269, in start
    self.loop.add_handler(self.stdout, self.handle_stdout, self.loop.READ)
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\site-packages\tornado\platform\asyncio.py", line 100, in add_handler
    self.asyncio_loop.add_reader(fd, self._handle_events, fd, IOLoop.READ)
  File "C:\Users\GCM\anaconda3\envs\MIT_807_P38\lib\asyncio\events.py", line 501, in add_reader
    raise NotImplementedError
NotImplementedError

When starting it using the clusters tab in jupyter then jupyter just freezes.

I am using everything through an anaconda environment on windows. (Not sure what else to include)

**Comment by minrk** [comment]
For now, try adding to your `ipython_config.py`:

```python
import asyncio
asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
```

I think this should help workaround the issue while we wait for a release with fixed compatibility with Windows + Python 3.8

**Comment by minrk** [comment]
This is fixed by tornado 6.1, which works on proactor
